.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 80,
+  "parser": "babylon",
+  "singleQuote": true
```

Comment:
Why don't we go with the default settings? I don't have any strong opinion on using single vs double quotes, but I definitely prefer using the default setting from a third party lib as long as it is safe.

Reply:
printWidth + parser are default and just added to show the settings, doubleQuote everywhere is the default, and as far as I can remember single quote is airbnb (so our eslint) default